services/minespace-web/src/components/Forms/noticeOfDeparture/EditNoticeOfDepartureForm.js

```diff
@@ -102,6 +105,15 @@ let EditNoticeOfDepartureForm = (props) => {
     setUploading(false);
   };
 
+  const handleWithdraw = () => {
+    delete noticeOfDeparture.nod_status;
```

Comment:
we are deleting this property?

Reply:
Good catch.  I was having trouble overriding this with `withdrawn` until I realized that property was being set based on the proponent's nod_type decision in `src/components/dashboard/mine/noticeOfDeparture/NoticeOfDeparture.js`.  I'll get rid of it.